src/isConnectedGraph.js

```diff
@@ -0,0 +1,37 @@
+export default function isConnectedGraph(graph = {}){
+  
+  if(graph == null){
+    return 'cannot check null'
+  }
+  
+  if(graph.constructor !== Object || graph.constructor === Array){
+    return 'use objects as input data'
+  }
+
+  const maxWalkingSteps = Object.keys(graph).length
+
+  for (let firstKey in graph){
+    for (let lastKey in graph){
+      if(firstKey != lastKey && !walkCheck(firstKey, lastKey, 1)){
+        return false
+      }
+    }
+    return true
```

Comment:
this return ensures you will never loop through the first `for` loop more than once. Are you sure you need that to be a loop?